portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase

```diff
@@ -19,58 +19,82 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could delete a Custom Export."
 	@priority = "4"
 	test DeleteCustomExportProcess {
-		LAR.exportSiteCP(
-			larFileName = "CustomExport",
-			siteScopeName = "Guest");
+		task ("Create a Custom Export") {
```

Comment:
Here you export just export the LAR file of a Site with some custom parameters, I think you could rather use something like: "Export the LAR file of the default site"